src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -359,7 +357,7 @@ void getRegionalHearingLocations_shouldReturnCorrespondingEpimsIdsForVenuesWithS
     }, nullValues = {"null"})
     void getHearingPriority(String isAdjournCase, String isUrgentCase, String expected) {
         // TODO Finish Test when method done
-        SscsCaseData caseData = SscsCaseData.builder()
+        caseData = SscsCaseData.builder()
             .urgentCase(isUrgentCase)
             .adjournCasePanelMembersExcluded(isAdjournCase)
             .build();
```

Comment:
you should set these by using toBuilder() or doing:
```suggestion
         caseData.setUrgentCase(isUrgentCase);
         caseData.setAdjournCasePanelMembersExcluded(isAdjournCase);
```

Reply:
can you also this this on the below, whichever you choose to do